subsys/net/lib/nrf_cloud/src/nrf_cloud_fota.c

```diff
@@ -926,3 +1031,103 @@ int nrf_cloud_fota_mqtt_evt_handler(const struct mqtt_evt *evt)
 
 	return 0;
 }
+
+#if IS_ENABLED(CONFIG_NRF_CLOUD_FOTA_BLE_DEVICES)
+int nrf_cloud_fota_ble_set_handler(nrf_cloud_fota_ble_callback_t cb)
+{
+	if (!cb) {
+		return -EINVAL;
+	}
+
+	ble_cb = cb;
+	return 0;
+}
+
+
+int nrf_cloud_fota_ble_update_check(const bt_addr_t * const ble_id)
```

Comment:
I take it I should call `nrf_cloud_fota_ble_update_check()` to see if a device I'm connected to has an update?

Reply:
correct